webapp/get_job_feed.py

```diff
@@ -0,0 +1,60 @@
+import re
+
+
+from canonicalwebteam.http import CachedSession
+from html import unescape
+
+
+api_session = CachedSession(
+    fallback_cache_duration=300, file_cache_directory=".webcache"
+)
+
+
+base_url = "https://api.greenhouse.io/v1/boards/Canonical/jobs"
+
+
+def get_job_feed(endpoint):
+    return api_session.get(f"{base_url}{endpoint}")
+
+
+def get_vacancies(department):
+    feed = get_job_feed("?content=true").json()
+    path_department = remove_special_chars(department)
```

Comment:
Why is `remove_special_chars` necessary? What cases are we controlling for here?

Reply:
I changed it to remove hyphens only.